Generals/Code/GameEngine/Source/GameClient/MessageStream/SelectionXlat.cpp

```diff
@@ -917,16 +917,8 @@ GameMessageDisposition SelectionTranslator::translateGameMessage(const GameMessa
 			ICoord2D pixel = msg->getArgument( 0 )->pixel;
 			UnsignedInt currentTime = (UnsignedInt) msg->getArgument( 2 )->integer;
 
-			Bool isClick = TheMouse->isClick(&m_deselectFeedbackAnchor, &pixel, m_lastClick, currentTime);
-
-			if (isClick &&
-					cameraPos.length() > TheMouse->m_dragTolerance3D)
```

Comment:
Mouse.ini:

```
  DragTolerance = 25                                ; How many pixels should we allow before it is a drag?
  DragTolerance3D = 25                              ; How many feet in worldspace should we allow before it is a drag?
  DragToleranceMS = 250                             ; if the mouse is held down for this long, we consider it a drag, not a click.
```

This is the only user of `m_dragTolerance3D`. So we can remove that field as well?

Perhaps the `cameraPos.length() > TheMouse->m_dragTolerance3D` was meant to be inside `isClick` function? It looks to be intentional, so I wonder if there is something to it we are missing.

Reply:
Also, judging by the descriptions, it appears to be redundant to `DragTolerance`.